apps/github/src/index.ts

```diff
@@ -35,6 +35,7 @@ const GenerateOptionsSchema = z.object({
   rapier: z.boolean().optional(),
   uikit: z.boolean().optional(),
   xr: z.boolean().optional(),
+  triplex: z.boolean().optional(),
```

Comment:
Let me know if it's possible to test the GitHub creation flow locally, otherwise will have to wait and see...